ultrahdr_api.h

```diff
@@ -372,7 +379,7 @@ UHDR_EXTERN uhdr_error_info_t uhdr_enc_set_gainmap_scale_factor(uhdr_codec_priva
  * for gamma correction of all planes separately. Default gamma value is 1.0.
  *
  * \param[in]  enc  encoder instance.
- * \param[in]  gamma  gamma of gainmap image. Any positive real number.
+ * \param[in]  gamma  gamma of gainmap image. Any positive real number > 0.0.
```

Comment:
Minor suggestion: since we already said "positive" then ">0.0" seems redundant.

Reply:
Yes, will take care of this in future cls